src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -967,6 +982,17 @@ namespace AirflowNetworkBalanceManager {
 				ErrorsFound = true;
 				SimObjectError = true;
 			}
+			for ( k = 1; k <= NumOfNodes; ++k ) {
+				if ( Node( k ).IsLocalNode ) {
+					ShowSevereError( RoutineName + "Invalid " + cAlphaFields( 3 ) + "=" + Alphas( 3 ) );
+					ShowContinueError( "A local air node is defined to INPUT the wind pressure coefficient curve, while Wind Pressure Coefficient Type is set to SurfaceAverageCalculation.");
+					ShowContinueError( "It requires  the Wind Pressure Coefficient Type be set to INPUT to use the local air node.");
+					ErrorsFound = true;
+					SimObjectError = true;
+					break;
+				}
+			}
```

Comment:
@jasondegraw I mean this *seems* OK to me.  Inside the `SurfaceAverageCalculation` block, looping over all nodes and throwing if any of them are local nodes.  That's the gist of what error you were wanting, right?

Reply:
@Myoldmopar I guess, but it still doesn't look quite right to me. If it looks OK to you, it's probably OK, though, so I'll approve here in a second.

Reply:
Thanks @jasondegraw, I think the only issue holding this up now is the unit test failure.